Makefile

```diff
@@ -18,9 +18,13 @@ CGO_ENABLED=0
 GOOS=linux
 
 build:
-	./hack/build.sh
+	./hack/build.sh -f
 .PHONY: build
 
+test-unit:
+	go test -v ./pkg/...
```

Comment:
good for now, but I can also add teh semantics of `build.sh --test` to run only tests when given alone.

Reply:
I was thinking same, we need a way only run tests.